examples/reference_environments/snowflake/jupyter.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM jupyter/minimal-notebook
+
+WORKDIR /gx
+
+COPY ./snowflake_example.ipynb ./
+
+RUN pip install great_expectations[snowflake, sqlalchemy-less-than-2]
```

Comment:
Nice! Mind adding a note here as to why sqlalchemy-less-than-2 is used for snowflake?

Reply:
done!